docs/UserGuide.md

```diff
@@ -35,21 +35,21 @@ title: User Guide
 
 </div>
 
-| Command    | Format                                   |
-| ---------- | ---------------------------------------- |
-| `help`     | `help`                                   |
-| `add`      | `add n/NAME [p/PHONE_NUMBER][e/EMAIL] [a/ADDRESS][c/CLASS_NAME...]` |
+| Command    | Format                                                                       |
+|------------|------------------------------------------------------------------------------|
+| `help`     | `help`                                                                       |
+| `add`      | `add n/NAME [p/PHONE_NUMBER][e/EMAIL] [a/ADDRESS][c/CLASS_NAME...]`          |
 | `edit`     | `edit INDEX [n/NAME][p/PHONE_NUMBER] [e/EMAIL][a/ADDRESS] [c/CLASS_NAME...]` |
-| `delete`   | `delete INDEX`                           |
-| `find`     | `find KEYWORD [MORE_KEYWORDS...]`        |
-| `list`     | `list`                                   |
-| `addc`     | `addc c/CLASS_NAME`                      |
-| `deletec`  | `deletec c/CLASS_NAME`                   |
-| `assign`   | `assign INDEX... c/CLASS_NAME`           |
-| `unassign` | `unassign INDEX... c/CLASS_NAME`         |
-| `listc`    | `listc`                                  |
-| `exit`     | `exit`                                   |
-| `class`    | `class c/CLASS_NAME`                     |
+| `delete`   | `delete INDEX`                                                               |
+| `find`     | `find KEYWORD [MORE_KEYWORDS...]`                                            |
+| `list`     | `list`                                                                       |
+| `addc`     | `addc c/CLASS_NAME`                                                          |
+| `deletec`  | `deletec c/CLASS_NAME`                                                       |
```

Comment:
```suggestion
| `deletec`  | `deletec c/CLASS_NAME...`                                                       |
```
Since now multiple classes can be deleted at once

Reply:
Actually this change is required at multiple places in the UG. We may fix it later, closer to finalizing the UG.

Reply:
yup, sure thing